lib/index.js

```diff
@@ -33,6 +33,25 @@ function getModuleIdentifier(seen, modulePath) {
   return `__${identifier}__`;
 }
 
+function shouldIgnore(pathParts) {
+  // ignore any of these extensions (.md, .html, etc)
+  if (IGNORED_EXTENSIONS.indexOf(pathParts.ext) > -1) {
+    return true;
+  }
+
+  // ignore .d.ts files
+  if (pathParts.base.slice(-5) === '.d.ts') {
```

Comment:
@dgeb - Unfortunately, the extension is actually `.ts` (both a path.parse and substring + lastIndexOf would agree).